src/components/Home.js

```diff
@@ -1,21 +1,26 @@
+import { useToast } from '@1hive/1hive-ui'
```

Comment:
Let's place this import after `styled-components` import.

We usually have imports from our own libraries after general library imports.